src/tools/web/client/src/components/Configuration.jsx

```diff
@@ -118,7 +127,7 @@ const Configuration = ({
     getKey: getCorrectKey,
     setKey: setCorrectKey,
     deleteKey: deleteCorrectKey,
-  }, tree)
+  }, { 'user': (tree && tree.user) || {} }) // only show user/ namespace
```

Comment:
> I decided that elektra-web should focus on the editing of user configuration values.

Yes, the focus is okay, but this `user` restriction seems to be arbitrary and not a focus? Most configuration admins care about lives in `system`. Or is this only as discussed with @tom-wa for PK? Let us discuss it via email.

When limiting scope, we should rather think about removing the cluster features?

> I had an issue where validation wouldn't run, because I was running everything on the namespace (and not with / prefix). That's why I decided to only show the user/ namespace and then prefix all set operations with / to make the validation run.

You can prefix everything with /. Then you do not need to care in which namespace the config is (except for creating new keys; here it would suffice for now that you can select the parameter of `-N`).

> I also don't think that editing metadata/specifications should be the focus of elektra-web. The idea is to have an existing configuration/specification and edit values there.

That is quite a limitation, then admins could not even modify descriptions or comments? For this PR meta data editing is not needed.

> While editing of metadata could be easily added later (the API is still there, I'm just only using user/ and /), the focus of this project should be the interactive configuration tree.

Metadata is an important part of the interactive configuration tree. You should update the use cases to better reflect your focus.